onnxscript/main.py

```diff
@@ -104,6 +104,54 @@ def transform(f):
     return transform
 
 
+def graph(parent: values.OnnxFunction):
+    """A parametric decorator used to annotate nested-functions that are used
+    as graph-attributes.
+
+    Args:
+        parent: Outer-level function.
+
+    Returns:
+        A decorator that returns its input function, but attaches a graph_proto
+        attribute representing the input function. The translation is not
+        done at this time, but previously when the outer-level function
+        was translated to an OnnxFunction. The decorator just looks up
+        and retrieves the GraphProto representation previously generated.
+
+    Example:
+
+    ::
```

Comment:
Possible to use backticks for code blocks like markdown? I think sphinx supports it.

Reply:
I am not sure. I just used the format from example above (which I assume works). Happy to leave it to the experts to fix (or let me know what exactly is required.)

Reply:
Tested; markdown doesn't work yet. reST is the way to go I think

Reply:
You can use extension .. exec_code:: to execute the example and dispaly the output in the documentation. The documentation fails if the example is no longer valid.

Reply:
I think I tried something like exec_code in the documentation (tutorial) and ran into the issue that our scripts don't work in that mode since we use inspect to get the source. Our documentation file examples are tested using our own test-case-driver (as you know). So, I will leave this as is for now. I am happy to update things if and when we have a answer to that.